scripts/build_utils.sh

```diff
@@ -601,7 +606,12 @@ setup_pbuilder() {
     fi
 
     if [ $FLAVOR = "crimson" ]; then
-        extrapackages='EXTRAPACKAGES="libc-ares-dev libcrypto++-dev libgnutls28-dev libhwloc-dev libnuma-dev libpciaccess-dev libprotobuf-dev libsctp-dev libyaml-cpp-dev protobuf-compiler ragel systemtap-sdt-dev"'
+        extrapackages='EXTRAPACKAGES="libc-ares-dev libcrypto++-dev libgnutls28-dev libhwloc-dev libnuma-dev libpciaccess-dev libprotobuf-dev libsctp-dev libyaml-cpp-dev protobuf-compiler ragel systemtap-sdt-dev software-properties-common curl"'
+        echo "$extrapackages" >> ~/.pbuilderrc
+    fi
+
+    if [ $FLAVOR = "default" ]; then
+        extrapackages='EXTRAPACKAGES="software-properties-common curl"'
```

Comment:
i'd suggest avoid repeating `"software-properties-common curl"` at multiple places. that's error prune and difficult to maintain.

and i'd suggest avoid hardwire to "default" here. as **all** flavors should have these packages installed.